test/helpers/check_slurm_output.sh

```diff
@@ -0,0 +1,14 @@
+# Checks whether a slurm output contains any errors
+SLURM_OUTPUT="$1"
+if ! grep -q "Model successfully built!" $SLURM_OUTPUT
+then
+    cat $SLURM_OUTPUT
+    echo "Model has not been successfully built"
+    exit 1
+fi
+if grep -q "CommandNotFoundError" $SLURM_OUTPUT
+then
+    cat $SLURM_OUTPUT
+    echo "CommandNotFoundError fount in slurm output. This is likely due to conda not being found."
+    exit 1
+fi
```

Comment:
I think you have some issue where your editor isnt putting newline at the end of file.

Reply:
Solved